substrate/frame/meta-tx/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
```

Comment:
This whole crate is great, and should live in `frame/examples` if it remains in any form.